### PR TITLE
Fix tests to expect ContainerKojiBuild as a part of KojiBuild and Bug Fixes

### PR DESCRIPTION
**Pull request by yashvardhannanavati**

### CODE REVIEW

File: purview/models/__init__.py [hunk 1]
```diff
@@ -19,6 +19,8 @@
 
 # Overrides from https://github.com/neo4j-contrib/neomodel/pull/327
 # These should be removed once a new version is released
+
```
Comment (mprahl) [inline purview/models/__init__.py]: This seems unnecessary.

Comment (yashvardhannanavati) [reply purview/models/__init__.py]: It was throwing a linting error :(

File: purview/utils/general.py [hunk 2]
```diff
@@ -66,14 +66,20 @@ def inflate_node(result):
...
     from purview.models import names_to_model
 
-    for label in result.labels:
-        if label in names_to_model:
-            node_model = names_to_model[label]
-            break
+    if 'ContainerKojiBuild' in result.labels:
+        result_label = 'ContainerKojiBuild'
+    elif len(result.labels) > 1:
+        raise RuntimeError('inflate_node encounted a node with the multiple labels: {0}. '
```
Comment (mprahl) [inline purview/utils/general.py]: "with the" => "with"

File: purview/models/__init__.py (hunk 1 above)
Comment (mprahl) [reply purview/models/__init__.py]: Oh alright. I think if you rebase develop on master and rebase your branch on develop this should be resolved.

**Comment by mprahl** [comment]
After the rebases occur and the typo is fixed. :+1:

**Pull request merged**